js/imports/request-methods/delete.mjs

```diff
@@ -0,0 +1,18 @@
+export async function deleteMethod(postId) {
+    try {
+        const accessToken = localStorage.getItem('accessToken');
+        const deleteData = {
+            method: 'DELETE',
+            headers: {
+                'Content-Type': 'application/json',
+                Authorization: `Bearer ${accessToken}`,
+            },
+        }
+        if(!accessToken) {
+            console.log("User is not authenticated!")
```

Comment:
Could you use an alert instead? Not every end user knows about the browser's console.